app/javascript/pages/listings/listing-detail.tsx

```diff
@@ -45,13 +50,33 @@ const ListingDetail = () => {
         {listing && (
           <article className="flex flex-wrap relative max-w-5xl m-auto w-full">
             <ListingDetailsImageCard listing={listing} />
```

Comment:
Lmk if there's a cleaner way to do this. I discovered that the applicate date component displays just below the image card if the listing is closed and under the pricing table if it's open.

Reply:
since this is the only place we're doing a mobile accommodation I think this is ok. if we find we keep having to add conditionals we can always come back to your idea for a higher level component

Reply:
I second that! @jacksongonzales